src/main/resources/feedbackQuestionContribResultStatsStudentViewTemplate.html

```diff
@@ -1,12 +1,14 @@
 <div class="panel panel-primary">
   <div class="panel-heading">
-    Comparison of work distribution ${contribAdditionalInfo}
+    Comparison of work distribution
   </div>
   <table class="table table-striped">
     <tbody>
       <tr>
         <td>
-          <strong>My view:</strong>
+         <span class="color-inherit" data-toggle="tooltip" data-placement="top"
+                 title="Compare 'my view' with 'team's view' to see if your perception is similar to team's perception">
```

Comment:
It's better to say "Compare 'My view' with 'Team's view'", so that the UI is consistent.
![image](https://user-images.githubusercontent.com/26651556/37835983-6334a68a-2eec-11e8-99d5-63047f6335aa.png)